vars/RedHat.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+karaf_java_home: /usr/lib/jvm/java-1.8.0-openjdk/
```

Comment:
Is the `/` here extraneous or should the same variable in the **Debian.yml** have one on its end? Might be nothing.

Reply:
It is probably extraneous. Both ubuntu and centos provisioned fine with the current settings. I can try removing it and testing centos again...